uk.co.compucorp.civicrm.hrleaveandabsences/js/src/leave-absences/leave-type-wizard/form/components/leave-type-wizard-form.component.js

```diff
@@ -12,7 +14,165 @@ define([], function () {
     }]
   };
 
-  function LeaveTypeWizardFormController ($log) {
+  function LeaveTypeWizardFormController ($log, sharedSettings) {
     $log.debug('Controller: LeaveTypeWizardFormController');
+
+    var vm = this;
+    var lineBreak = '\n\n';
+
+    vm.leaveTypeCategory = '';
+    vm.leaveTypeCategories = [
```

Comment:
is there any plan to fetch these from the backend at some point?

Reply:
No, I don't think so. At least not in the mid-term. But I was thinking of moving the categories map to a separate file.

Reply:
is this something that was decided in agreement with the backend dev?

Reply:
No, this was agreed during Wiki page reviewal.